src/js/apps/patients/worklist/worklist_sort.js

```diff
@@ -26,13 +26,13 @@ const SortOption = Backbone.Model.extend({
     const fieldName = this.get('field_name');
     const keys = this.get('field_key').split('.');
     const direction = this.get('direction');
-    const type = this.get('type');
+    const sortType = this.get('sort_type');
```

Comment:
Why `sortType` instead of just `sort` ?

Reply:
It's more descriptive when reading the code, in my opinion. Not a huge deal either way for me though.

Reply:
That's between you and @paulfalgout. To me it feels overly verbose.